### ISSUE TITLE
CQL fictions are added in coverage highlighting

### ISSUE THREAD

**Issue by adongare** [projecttacoma/fqm-execution#143]
Hello,

I think earlier, functions were not added to the coverage(highlighted HTML). After switching to [v1.0.0-beta.4](https://github.com/projecttacoma/fqm-execution/releases/tag/v1.0.0-beta.4), noticed that functions are being added to the HTML highlighting in red color(screenshot attached). I'm not sure if this is an intentional shift from the earlier approach. Even in cqm-execution, we used to exclude functions from coverage.

<img width="677" alt="Screen Shot 2022-10-07 at 11 30 36 AM" src="https://user-images.githubusercontent.com/23436747/194592178-144dca3d-729e-4dd9-a4a3-4c77d0d8a0c3.png">

Also, in `detailedResults.clauseResults` functions are marked as `NA` however in `detailedResults.statementResults` thier relevance is TRUE/FALSE. I'm using clauseResults to calculate the overall coverage percentage for a measure group. So, this causes some mismatch in coverage percentage and highlighting. In the above screenshot, I've 100% coverage but CQL functions are highlighted in red color. Is this a potential bug or the way I calculate overall coverage for a measure group is incorrect?

Thanks!

**Comment by mgramigna** [comment]
Hi @adongare, thanks for the issue. 

We poked at this today, and it looks like functionality that we introduced in `v1.0.0-beta.3` incorrectly modified a boolean condition that caused functions to start highlighting as false instead of being left alone. I have pushed up #148 for review that should be merged and released soon that fixes this specific issue.

Along with the above release, we did include some logic for generating the total clause coverage for a population for a measure. To retrieve that HTML, you could do:

``` TypeScript
const { results, coverageHTML } = await calculate(measureBundle, patientBundles, {
  /* other options omitted */,
  calculateClauseCoverage: true
});
```

The logic that actually calculates the percentage for this is found here, if that helps: https://github.com/projecttacoma/fqm-execution/blob/3f98c2f9aecab1091b1c2d50dffb83e25019c848/src/calculation/HTMLBuilder.ts#L224

**Comment by mgramigna** [comment]
Fixed in [v1.0.0-beta.5](https://github.com/projecttacoma/fqm-execution/releases/tag/v1.0.0-beta.5) (now published to npm)

**Comment by adongare** [comment]
Thank you so much, @mgramigna for the fix and also, for the `calculateClauseCoverage` function. We'll go ahead and pull this version.
